WooCommerce/Classes/AppDelegate.swift

```diff
@@ -318,11 +320,26 @@ private extension AppDelegate {
     }
 
     func handleLaunchArguments() {
-        if ProcessInfo.processInfo.arguments.contains("logout-at-launch") {
+        if BuildConfiguration.shouldLogoutAtLaunch {
           ServiceLocator.stores.deauthenticate()
         }
 
-        if ProcessInfo.processInfo.arguments.contains("disable-animations") {
+        if BuildConfiguration.shouldUseScreenshotsNetworkLayer {
+            /// Print the location of the core data DB for debugging
+            if let storeUrl = ServiceLocator
+                .storageManager
+                .persistentContainer
```

Comment:
Works for me – definitely understand about isolating this!

I personally like printing the path the apps' documents directory or something on startup, because when debugging I like to open up the core data DB a bunch to see what it's doing. 

But for now, I'm fine to just drop this code to avoid further changes – I'll replace it some other future tooling thing probably 😜

Removed in 046c07d7a91a0dcb5acdd438a8f2909a31421c31